components/org.wso2.carbon.security.caas/src/main/java/org/wso2/carbon/security/caas/user/core/bean/Attribute.java

```diff
@@ -21,18 +21,9 @@
  */
 public class Attribute {
 
-    private String identityStoreId;
     private String attributeName;
     private String attributeValue;
 
-    public String getIdentityStoreId() {
-        return identityStoreId;
-    }
-
-    public void setIdentityStoreId(String identityStoreId) {
-        this.identityStoreId = identityStoreId;
-    }
-
```

Comment:
We're planning to return Attributes only from connectors. Therefore, when an attribute is returned from a connector we will know which connector returned that attribute. All the stores, claim managers .etc will return claims not attributes.